### PR TITLE
{lang}[foss/2023b,gfbf/2023b] R v4.4.1, R-bundle-CRAN v2024.06

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

* [x] #20893
* [x] #20894
* [x] #20892

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 44 out of 44 (2 easyconfigs in total)
bear-pg0210u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8480CL (sapphirerapids), Python 3.6.8
See https://gist.github.com/branfosj/2be49f1424519596a527ef5ab55a6df9 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 2 out of 5 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/0d9e15c48f7776ef6f11575f247698c7 for a full test report.

**Comment by verdurin** [comment]
That failure was down to inability to download `libopus`, which I fixed manually.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/1164113fd95c764fc263b21440652acd for a full test report.

File: easybuild/easyconfigs/r/R-bundle-CRAN/R-bundle-CRAN-2024.06-foss-2023b.eb [hunk 1]
```diff
@@ -0,0 +1,3430 @@
...
+    }),
+    ('readbitmap', '0.1.5', {
+        'checksums': ['737d7d585eb33de2c200da64d16781e3c9522400fe2af352e1460c6a402a0291'],
+    }),
+    ('imager', '1.0.2', {
+        'checksums': ['7c849086cb17d6c5aefc106217363e14afbcda2a9e0120687d40805b5e1c566a'],
+    }),
+    ('signal', '1.8-0', {
+        'checksums': ['89cba854167a2b051a58cf3b73ccbf74eeb47c890ac39720611cd41f86b94684'],
+    }),
```
Comment (branfosj) [inline easybuild/easyconfigs/r/R-bundle-CRAN/R-bundle-CRAN-2024.06-foss-2023b.eb:1596]:
```suggestion
    ('signal', '1.8-0', {
        'checksums': [
            ('89cba854167a2b051a58cf3b73ccbf74eeb47c890ac39720611cd41f86b94684',
             '0a604949bae91410a150a22cfa02d954f5b83166cc7a73e5409554d00e0417a7'),
        ],
    }),
```

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen3

**Comment by verdurin** [comment]
Hmm, checksum problem with `pan`?

File: easybuild/easyconfigs/r/R-bundle-CRAN/R-bundle-CRAN-2024.06-foss-2023b.eb [hunk 2]
```diff
@@ -0,0 +1,3433 @@
...
+        'checksums': ['0fa65a5fe0a46cffe221b7696b52adb82dd4d7a692a895484e438e439594e10a'],
+    }),
+    ('forcats', '1.0.0', {
+        'checksums': ['c5bb157909d92e1e1a427c0dc5cb358ea00a43a14918a9088fa4f6630962254e'],
+    }),
+    ('haven', '2.5.4', {
+        'checksums': ['9e1531bb37aa474abd91db5e0ed9e3a355c03faa65f4e653b3ea68b7c61ea835'],
+    }),
+    ('pan', '1.9', {
+        'checksums': ['cd91232d653783ea7f34c0eebaa80c472b5501b21eea500c4c1a8e57116c6eea'],
```
Comment (branfosj) [inline easybuild/easyconfigs/r/R-bundle-CRAN/R-bundle-CRAN-2024.06-foss-2023b.eb:583]:
```suggestion
        'checksums': [
            ('e37e184c3c1b7a34f54dd95335e6bc730fd5716d2d2dc20c24279401aa673c52',
             'cd91232d653783ea7f34c0eebaa80c472b5501b21eea500c4c1a8e57116c6eea'),
        ],
```

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen3

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.12
See https://gist.github.com/bedroge/d564296e91f2141f5dadca6fd551ce72 for a full test report.

**Comment by bedroge** [comment]
@branfosj I'm testing this with the EESSI bot (see https://github.com/EESSI/software-layer/pull/644), and it successfully built this for the x86_64 targets, but it's failing for aarch64 due to an issue that was fixed for the previous version in https://github.com/easybuilders/easybuild-easyconfigs/pull/20238. Could you please include that same fix for this version?

**Comment by branfosj** [comment]
> @branfosj I'm testing this with the EESSI bot (see [EESSI/software-layer#644](https://github.com/EESSI/software-layer/pull/644)), and it successfully built this for the x86_64 targets, but it's failing for aarch64 due to an issue that was fixed for the previous version in #20238. Could you please include that same fix for this version?

Done in 12ef0f1

**Comment by bedroge** [comment]
I had to work around some issues with Highway (not related to this PR, but it's a dependency) on Arm, but successful test reports/builds with the EESSI bot can now be found in https://github.com/EESSI/software-layer/pull/644 for `x86_64/generic, x86_64/intel/haswell, x86_64/intel/skylake_avx512, x86_64/amd/zen2, x86_64/amd/zen3, aarch64/generic, aarch64/neoverse_n1, aarch64/neoverse_v1`.
I'll try to do some more builds on our local cluster and upload the test reports here.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 63 out of 70 (2 easyconfigs in total)
easybuild-el8.cloud.in.bmrc.ox.ac.uk - Linux Rocky Linux 8.10, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/b4ef6771126ca1b4b4ba701421dc2105 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-el8.cloud.in.bmrc.ox.ac.uk - Linux Rocky Linux 8.10, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/5bc6e55f98a529db2066135e36372bf7 for a full test report.

**Comment by verdurin** [comment]
@bedroge this seems good to me now, but happy to wait for your additional builds.

**Comment by bedroge** [comment]
> @bedroge this seems good to me now, but happy to wait for your additional builds.

Those build jobs apparently had some issues unrelated to this PR. Since it has been tested thoroughly on other systems, also with the EESSI bot, I'm also perfectly fine with merging this now. I can still upload those test reports later, as I want to have this installed on our cluster anyway.

**Review by bedroge — APPROVED** [review]
Looks good to me.

**Comment by bedroge** [comment]
Going in, thanks @branfosj!

**Pull request merged**